test/test_buffer.py

```diff
@@ -3,6 +3,10 @@
 from neovim.compat import IS_PYTHON3
 
 
+def test_repr(vim):
+    assert repr(vim.current.buffer) == "<Buffer(handle=2)>"
```

Comment:
hmm, looks a bit fragile, but test run order seems deterministic. So guess we can keep this for now and change it to a regex later if it turns out to be too much churn.